packages/smooth_app/lib/generic_lib/design_constants.dart

```diff
@@ -17,7 +17,7 @@ const double MINIMUM_TOUCH_SIZE = kMinInteractiveDimension;
 const double DEFAULT_ICON_SIZE = 24.0;
 
 /// Background, e.g SmoothCard
-const Radius ROUNDED_RADIUS = Radius.circular(20.0);
+const Radius ROUNDED_RADIUS = Radius.circular(10.0);
```

Comment:
By changing this button, it will have an impact on the whole app.
Please provide specific radiuses when you need it.

Reply:
Okay, in Figma the corners looked less than 20px, 
However, I will revert back to the original.